src/config.ts

```diff
@@ -65,3 +65,8 @@ function getConfig(confType: string) : Config {
 }
 
 export const config : Config = getConfig(process.env.NODE_ENV || dev.conf_type);
+const mongoHost = process.env.MONGO_HOST || config.db.host;
```

Comment:
Remove `mongoHost` OR `connectionString` and export `process.env.MONGO_HOST || 'mongodb://localhost:27017';`.